libraries/aws_iam_groups.rb

```diff
@@ -37,10 +39,13 @@ def fetch_data
         return [] if !groups || groups.empty?
 
         groups.groups.each do |g|
-          group_rows += [{ group_name: g.group_name,
-                           group_id:   g.group_id,
-                           arn:        g.arn,
-                           users:      @aws.iam_client.get_group({ 'group_name': g.group_name }).users.map(&:user_name) }]
+          inline_policies = @aws.iam_client.list_group_policies({ 'group_name': g.group_name }).policy_names
+          group_rows += [{ group_name:          g.group_name,
+                           group_id:            g.group_id,
+                           arn:                 g.arn,
+                           users:               @aws.iam_client.get_group({ 'group_name': g.group_name }).users.map(&:user_name),
+                           has_inline_policies: !inline_policies.empty?,
+                           inline_policy_names: @aws.iam_client.list_group_policies({ 'group_name': g.group_name }).policy_names }]
```

Comment:
isn't this same as `inline_policies` ?

```suggestion
                           inline_policy_names: inline_policies }]
```

Reply:
Tested and confirmed this line is not required. Thanks for the catch.